core/src/main/scala/cats/Invariant.scala

```diff
@@ -37,7 +37,7 @@ import scala.util.control.TailCalls.TailRec
 
   /**
    * Transform an `F[A]` into an `F[B]` by providing a transformation from `A`
-   * to `B` and one from `B` to `A`.
+   * to `B` and its inverse from `B` to `A` i.e. `f.andThen(g) == identity`.
```

Comment:
@jmgimeno informs me that the category-theoretic term here is "retraction".
https://en.wikipedia.org/wiki/Section_(category_theory)